tests/Aspire.Workload.Tests/StarterTemplateRunTestsBase.cs

```diff
@@ -103,7 +103,7 @@ static async Task<List<string[]>> GetAndValidateCellTexts(ILocator tableLoc)
                     r => Assert.True(DateTime.TryParse(r, out _)),
                     r => Assert.True(int.TryParse(r, out var actualTempC) && actualTempC >= -20 && actualTempC <= 55),
                     r => Assert.True(int.TryParse(r, out var actualTempF) && actualTempF >= -5 && actualTempF <= 133),
-                    r => Assert.Contains(r, ["Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"]));
+                    r => Assert.Contains(r, new HashSet<string>{"Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"}));
```

Comment:
Still curious why this didn't work with the collection initializer, but oh well, no need to change it.

Reply:
`error CS0121: The call is ambiguous between the following methods or
properties: 'Assert.Contains<T>(T, HashSet<T>)' and
'Assert.Contains<T>(T, SortedSet<T>)'`